packages/runtime/src/runtimes/react/components/draft-switcher/draft-switcher.tsx

```diff
@@ -46,12 +48,15 @@ export function DraftSwitcher({ isDraft }: { isDraft: boolean }) {
             : null}
         </span>
       )}
-      <script
+      {/* Insert draft mode information into the DOM to make it easier to debug draft mode-related 
+          issues on production sites */}
+      <Script
         type="application/json"
         id="makeswift-draft-info"
         dangerouslySetInnerHTML={{
           __html: JSON.stringify({ draft: isDraft, inBuilder: isInBuilder }),
         }}
+        strategy="afterInteractive"
```

Comment:
Note that one of the suggested solutions in the error message is to make the `<script>` tag async, which was my first preference since it'd avoid introducing yet another dependency on Next, but that did not resolve the issue.